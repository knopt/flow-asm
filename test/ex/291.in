3
16
0.547566808377427
-1.2642754698132812 -9.861921224490992 -9.285031334529867 
-7.566404820139514 4.024722098037355 2.0523439763956457 
-0.7938099928264428 -8.943348362445429 3.909786192637588 
4.71074861241525 -3.075184707491734 6.051576194279551 
7.955024517478936 -1.6409786692621893 1.8268781565672843 
4.475692455246978 6.582352019100348 1.7062760958212912 
8.620165243457286 -6.75418933465548 -4.6992250052096995 
-9.187401402317668 8.514606146023219 -0.31386875242380086 
-1.2414002186965973 -1.8923150432327809 -2.9543690489935166 
-3.6072466625483806 -4.423938957423104 -9.824334537237132 
2.1401890876095955 3.7180098644728155 -8.289536391099377 
0.6443013783852969 9.16607516059209 8.692840875338504 
4.6349145798124205 -4.785061960563171 -5.243092697408218 
9.224308040245678 -1.6908378508963189 -7.4581844003977915 
-2.8604691552694383 -4.009960799708299 7.167052539544546 
1.774187382020747 -5.800577013040522 -7.977876355565567 

-7.903825849654133 -6.167340012325601 3.79068889472396 0.47470445331061484 3.364513943706589 -5.590103110952382 -8.252491159098952 -6.444112031719415 -2.373137679280301 -0.29193462432693806 5.212072704527127 6.330641028067291 8.277523473427465 7.977410202549535 0.9049531114549261 3.5668098978573433 
