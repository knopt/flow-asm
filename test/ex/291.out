-11.035465137515713 3.706677775258939 3.8950325313807763 
6.392968621433843 -22.561491945901583 -14.60327626900204 
-1.225595391832718 14.063663831372478 -6.734267399519457 
-0.08737193610340022 6.049227462333451 -14.854923376093502 
-9.753048549400855 14.158254606084903 2.094994326211225 
-4.439435956318457 -4.509186375530023 -5.530664343622605 
-28.66871905907408 22.144394289694624 13.497132531056208 
10.659722199734794 -20.521860468416484 -3.7176253600827476 
-9.836464038950513 -2.1572310841607814 0.7866041555659469 
8.155595880483908 7.204700016092259 9.493561333133794 
0.818849262904007 -4.315161709512285 18.41876533943507 
13.443037051188634 -12.450779814209575 -18.081977102416467 
9.71612168050963 20.350483749192918 11.260732609651821 
-5.662504667123136 4.145085364994723 8.272874063940076 
17.810185751676464 7.322709326600135 -27.205193969369045 
-0.2579799516726662 0.937500031902454 3.679899731333566 
